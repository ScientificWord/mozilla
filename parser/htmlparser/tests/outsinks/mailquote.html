<html>
<head><!-- ***** BEGIN LICENSE BLOCK *****
   - Version: MPL 1.1/GPL 2.0/LGPL 2.1
   -
   - The contents of this file are subject to the Mozilla Public License Version
   - 1.1 (the "License"); you may not use this file except in compliance with
   - the License. You may obtain a copy of the License at
   - http://www.mozilla.org/MPL/
   -
   - Software distributed under the License is distributed on an "AS IS" basis,
   - WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License
   - for the specific language governing rights and limitations under the
   - License.
   -
   - The Original Code is Mozilla Communicator client code.
   -
   - The Initial Developer of the Original Code is Netscape
   - Communications Corporation. Portions created by Netscape are
   - Copyright (C) 1998-1999 Netscape Communications Corporation. All
   - Rights Reserved.
   -
   - Contributor(s):
   -
   - Alternatively, the contents of this file may be used under the terms of
   - either the GNU General Public License Version 2 or later (the "GPL"), or
   - the GNU Lesser General Public License Version 2.1 or later (the "LGPL"),
   - in which case the provisions of the GPL or the LGPL are applicable instead
   - of those above. If you wish to allow use of your version of this file only
   - under the terms of either the GPL or the LGPL, and not to allow others to
   - use your version of this file under the terms of the MPL, indicate your
   - decision by deleting the provisions above and replace them with the notice
   - and other provisions required by the LGPL or the GPL. If you do not delete
   - the provisions above, a recipient may use your version of this file under
   - the terms of any one of the MPL, the GPL or the LGPL.
   -
   - ***** END LICENSE BLOCK ***** -->
<title>Mail Quoting Test</title>
</head>

<body>

This page is a test of mail quoting.
<p>

I hope you will enjoy these quotes from <em>Hamlet</em>, introduced by a fairly long line to see how quotations get wrapped:
<p>

<blockquote type="cite">
<blockquote type="cite">
<em>(These have <b>br</b> tags after them.</em><br>
To be, or not to be, that is the question<br>
Whether 'tis nobler in the mind to suffer<br>
The slings and fortunes of outrageous fortune<br>
Or to take arms against a sea of troubles,
And by opposing end them.<br>
</blockquote>

<p>
Oh, what a mind is here o'erthrown.<br>

<blockquote type="cite">
<em>(The next line does not end with a <b>br</b> tag.).</em><br>
Oh, what a rogue and peasant slave am I.
</blockquote>

<em>(Neither does the next line:)</em></br>
The observed of all observers, quite, quite down!

</blockquote>

<p>
Now we're outside all blockquotes.
</p>

<p>
Now let's test some more complicated quotes:<br>
</p>

On 02/22/2003 06:29 AM, Koike Kazuhiko wrote:
<br><blockquote type=cite>abc
<br></blockquote>def
<br>
<br>

On 02/23/2003 06:49 AM, Blake Kaplan wrote:
<br><blockquote type=cite>On 02/22/2003 06:29 AM, Koike Kazuhiko wrote:
<br><blockquote type=cite>abc
<br></blockquote>def
<br></blockquote>ghi
<br><br>

The alternate method of quoting:<br><br>

<pre wrap>On 02/22/2003 06:29 AM, Koike Kazuhiko wrote:
</pre><blockquote type=cite><pre wrap>abc
</pre></blockquote><pre wrap><!---->def

</pre></body>
</body>
</html>
