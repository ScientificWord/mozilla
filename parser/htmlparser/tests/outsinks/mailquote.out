This page is a test of mail quoting.

I hope you will enjoy these quotes from /Hamlet/,
introduced by a fairly long line to see how
quotations get wrapped:

>> /(These have *br* tags after them./
>> To be, or not to be, that is the question
>> Whether 'tis nobler in the mind to suffer
>> The slings and fortunes of outrageous fortune
>> Or to take arms against a sea of troubles, And
>> by opposing end them.
>
> Oh, what a mind is here o'erthrown.
>
>> /(The next line does not end with a *br* tag.)./
>> Oh, what a rogue and peasant slave am I. 
> /(Neither does the next line:)/
> The observed of all observers, quite, quite down! 

Now we're outside all blockquotes.

Now let's test some more complicated quotes:

On 02/22/2003 06:29 AM, Koike Kazuhiko wrote:
> abc
def

On 02/23/2003 06:49 AM, Blake Kaplan wrote:
> On 02/22/2003 06:29 AM, Koike Kazuhiko wrote:
>> abc
> def
ghi

The alternate method of quoting:

On 02/22/2003 06:29 AM, Koike Kazuhiko wrote:
> abc
def

